rero_ils/modules/notifications/dispatcher.py

```diff
@@ -155,7 +155,7 @@ def _create_email(recipients, reply_to, ctx_data, template):
             template_body=template,
             sender=current_app.config.get('DEFAULT_SENDER_EMAIL',
                                           'noreply@rero.ch'),
-            reply_to=reply_to,
+            reply_to=','.join(reply_to),  # the client is unable to manage list
```

Comment:
I'am not sure this is correct. Excample:
`['person1@gmail.com', 'person2@gmail.com']` after the join we will have a string with `'person1@gmail.com, person2@gmail.com'`

Reply:
@rerowep 
According to RFC5322 : "In either case, an optional reply-to field MAY also be included, which contains the field name "Reply-To" and a comma-separated list of one or more addresses."

So it seems correct for me. It should better to use list, but client (flask-mail) works fine with list for recipients, but not for reply-to :(